### ISSUE TITLE
Autotoggle

### ISSUE THREAD

**Issue by UlikGames** [MarlBurroW/RLMarlbot#36]
Just a idea. I personally toggle off and on ingame mixed it with my play (aerial or other things) with autotoggle it keeps toggling all round u cant disable. I prefer to toggle on start of game or it is good we can disable in middle of game

**Comment by MarlBurroW** [comment]
I agree, you should be able to disable the bot in autotoggle mode, but what is the behavior when you have disabled the bot ? Does the bot should be automatically re-toggled on the next match ? Or the bot remains disabled and wait the next manual activation ?
Need more users opinion about that.

**Comment by UlikGames** [comment]
I prefer toggle on every round. Cause I play with bot normally. Mostly disables it and continue play normally. And a little idea don't know how to do it but you can add input recognization like when we push any button on keyboard or mouse bot just stop. When we release it again bot starts again. But it just a idea. Both okey for me.

**Comment by typoEXE** [comment]
> I prefer toggle on every round. Cause I play with bot normally. Mostly disables it and continue play normally. And a little idea don't know how to do it but you can add input recognization like when we push any button on keyboard or mouse bot just stop. When we release it again bot starts again. But it just a idea. Both okey for me.

I know it is possible to make it where if there is any type of feedback from the keyboard/mouse, it will automatically disable the bot. I personally would like this feature as well. Useful when I'm trying to go for a freestyle. When I have free time I'll try to implement it but I have a feeling it will mess with the current toggle features (especially auto toggle). Marl may have to be the one to look into this feature after he takes care of the more prioritized tasks. I know we can simply just toggle the bot off and on, so this would be considered a quality of life enhancement.

**Comment by psxjtwr01** [comment]
> > I prefer toggle on every round. Cause I play with bot normally. Mostly disables it and continue play normally. And a little idea don't know how to do it but you can add input recognization like when we push any button on keyboard or mouse bot just stop. When we release it again bot starts again. But it just a idea. Both okey for me.
> 
> I know it is possible to make it where if there is any type of feedback from the keyboard/mouse, it will automatically disable the bot. I personally would like this feature as well. Useful when I'm trying to go for a freestyle. When I have free time I'll try to implement it but I have a feeling it will mess with the current toggle features (especially auto toggle). Marl may have to be the one to look into this feature after he takes care of the more prioritized tasks. I know we can simply just toggle the bot off and on, so this would be considered a quality of life enhancement.

What I would do is edit the input handler cuz all it does RN is checks for the toggle key but if you change it to add a boost button kind of like tas. It would require one more config option but it's not that bad
